src/nft/components/profile/list/Modal/ListModalSection.tsx

```diff
@@ -109,7 +109,7 @@ export const ListModalSection = ({ sectionType, active, content, toggleSection }
             {isCollectionApprovalSection ? (
```

Comment:
if there's no `uniqueCollections` to approve we should just hide the section so `isCollectionApprovalSection && uniqueCollections !== undefined`

Reply:
out-of-scope?

Reply:
it's relevant given the newly added `?? 1` to Plural but a nit and non-blocking. If you could add a ticket to the backlog so it's not forgotten

Reply:
https://linear.app/uniswap/issue/WEB-2615/hide-nft-approval-section-if-there-are-no-collections-to-approve